<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BookAppointment</class>
 <widget class="QDialog" name="BookAppointment">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>696</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MedPlus - Book Appointment</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
        stop:0 #f5f7fa, stop:1 #c3cfe2);
}</string>
  </property>
  <widget class="QFrame" name="frameCircle1">
   <property name="geometry">
    <rect>
     <x>-80</x>
     <y>-80</y>
     <width>300</width>
     <height>300</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame {
    background: qradialgradient(cx:0.5, cy:0.5, radius:1,
        fx:0.5, fy:0.5, stop:0 rgba(33, 147, 176, 0.2), stop:1 transparent);
    border-radius: 150px;
    border: none;
}</string>
   </property>
  </widget>
  <widget class="QFrame" name="frameCircle2">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>500</y>
     <width>350</width>
     <height>350</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame {
    background: qradialgradient(cx:0.5, cy:0.5, radius:1,
        fx:0.5, fy:0.5, stop:0 rgba(109, 213, 237, 0.2), stop:1 transparent);
    border-radius: 175px;
    border: none;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>50</y>
     <width>500</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-size: 36px;
    font-weight: bold;
    color: #ffffff;
    background: qlineargradient(x1:0, y1:0, x2:1, y2:0,
        stop:0 #1976d2, stop:0.5 #42a5f5, stop:1 #64b5f6);
    padding: 20px 30px;
    border-radius: 15px;
    border: 3px solid rgba(255, 255, 255, 0.3);
    letter-spacing: 2px;
    qproperty-alignment: AlignCenter;
}</string>
   </property>
   <property name="text">
    <string>üìÖ Book Appointment</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblSubtitle">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>165</y>
     <width>500</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-size: 16px;
    color: #37474f;
    background: transparent;
    qproperty-alignment: AlignCenter;
    font-weight: 500;
}</string>
   </property>
   <property name="text">
    <string>Schedule your consultation with our expert doctors</string>
   </property>
  </widget>
  <widget class="QFrame" name="frameForm">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>230</y>
     <width>500</width>
     <height>380</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame {
    background-color: rgba(255, 255, 255, 0.95);
    border-radius: 20px;
    border: 2px solid rgba(25, 118, 210, 0.3);
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>270</y>
     <width>150</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-size: 18px;
    font-weight: 600;
    color: #1565c0;
    background: transparent;
    padding: 4px;
    letter-spacing: 0.5px;
}</string>
   </property>
   <property name="text">
    <string>üë®‚Äç‚öïÔ∏è Doctor:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboDoctor">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>270</y>
     <width>240</width>
     <height>45</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
    background-color: white;
    border: 2px solid #1976d2;
    border-radius: 10px;
    padding: 8px 12px;
    font-size: 15px;
    font-weight: 500;
    color: #2c3e50;
}

QComboBox:hover {
    border: 2px solid #1565c0;
    background-color: #f5f9ff;
}

QComboBox::drop-down {
    border-left: 2px solid #1976d2;
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
        stop:0 #42a5f5, stop:1 #1976d2);
    width: 30px;
    border-top-right-radius: 8px;
    border-bottom-right-radius: 8px;
}

QComboBox::down-arrow {
    width: 0;
    height: 0;
    border-left: 6px solid transparent;
    border-right: 6px solid transparent;
    border-top: 8px solid white;
    margin-right: 8px;
}

QComboBox QAbstractItemView {
    background-color: white;
    border: 2px solid #1976d2;
    border-radius: 8px;
    selection-background-color: #42a5f5;
    selection-color: white;
    padding: 5px;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>345</y>
     <width>150</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-size: 18px;
    font-weight: 600;
    color: #1565c0;
    background: transparent;
    padding: 4px;
    letter-spacing: 0.5px;
}</string>
   </property>
   <property name="text">
    <string>üìÜ Date:</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateAppointment">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>345</y>
     <width>241</width>
     <height>45</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QDateEdit {
    font-size: 15px;
    font-weight: 500;
    padding: 8px 12px;
    border: 2px solid #1976d2;
    border-radius: 10px;
    background-color: white;
    color: #2c3e50;
}

QDateEdit:hover {
    border: 2px solid #1565c0;
    background-color: #f5f9ff;
}

QDateEdit::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 30px;
    border-left: 2px solid #1976d2;
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
        stop:0 #42a5f5, stop:1 #1976d2);
    border-top-right-radius: 8px;
    border-bottom-right-radius: 8px;
}

QDateEdit::down-arrow {
    width: 0;
    height: 0;
    border-left: 6px solid transparent;
    border-right: 6px solid transparent;
    border-top: 8px solid white;
    margin-right: 8px;
}

QCalendarWidget {
    background-color: white;
    border: 2px solid #1976d2;
    border-radius: 10px;
}

QCalendarWidget QToolButton {
    color: white;
    background-color: #1976d2;
    border-radius: 5px;
}

QCalendarWidget QAbstractItemView:enabled {
    color: #2c3e50;
    background-color: white;
    selection-background-color: #42a5f5;
    selection-color: white;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>420</y>
     <width>150</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-size: 18px;
    font-weight: 600;
    color: #1565c0;
    background: transparent;
    padding: 4px;
    letter-spacing: 0.5px;
}</string>
   </property>
   <property name="text">
    <string>üïê Time Slot:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboSlot">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>420</y>
     <width>240</width>
     <height>45</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
    background-color: white;
    border: 2px solid #1976d2;
    border-radius: 10px;
    padding: 8px 12px;
    font-size: 15px;
    font-weight: 500;
    color: #2c3e50;
}

QComboBox:hover {
    border: 2px solid #1565c0;
    background-color: #f5f9ff;
}

QComboBox::drop-down {
    border-left: 2px solid #1976d2;
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
        stop:0 #42a5f5, stop:1 #1976d2);
    width: 30px;
    border-top-right-radius: 8px;
    border-bottom-right-radius: 8px;
}

QComboBox::down-arrow {
    width: 0;
    height: 0;
    border-left: 6px solid transparent;
    border-right: 6px solid transparent;
    border-top: 8px solid white;
    margin-right: 8px;
}

QComboBox QAbstractItemView {
    background-color: white;
    border: 2px solid #1976d2;
    border-radius: 8px;
    selection-background-color: #42a5f5;
    selection-color: white;
    padding: 5px;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblPatientInfo">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>495</y>
     <width>150</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-size: 18px;
    font-weight: 600;
    color: #1565c0;
    background: transparent;
    padding: 4px;
    letter-spacing: 0.5px;
}</string>
   </property>
   <property name="text">
    <string>üìù Notes:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditNotes">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>495</y>
     <width>240</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit {
    background-color: white;
    border: 2px solid #1976d2;
    border-radius: 10px;
    padding: 8px 12px;
    font-size: 14px;
    color: #2c3e50;
}

QTextEdit:hover {
    border: 2px solid #1565c0;
    background-color: #f5f9ff;
}

QTextEdit:focus {
    border: 2px solid #0d47a1;
}</string>
   </property>
   <property name="placeholderText">
    <string>Any specific concerns or symptoms...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBook">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>650</y>
     <width>171</width>
     <height>55</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    font-size: 18px;
    font-weight: bold;
    color: white;
    background: qlineargradient(x1:0, y1:0, x2:1, y2:0,
        stop:0 #1976d2, stop:1 #42a5f5);
    border: none;
    border-radius: 12px;
    padding: 12px 24px;
}

QPushButton:hover {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:0,
        stop:0 #1565c0, stop:1 #1976d2);
}

QPushButton:pressed {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:0,
        stop:0 #0d47a1, stop:1 #1565c0);
}</string>
   </property>
   <property name="text">
    <string>‚úì Book Now</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>650</y>
     <width>140</width>
     <height>55</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    font-size: 18px;
    font-weight: bold;
    color: #1976d2;
    background: transparent;
    border: 2px solid #1976d2;
    border-radius: 12px;
    padding: 12px 24px;
}

QPushButton:hover {
    background-color: #e3f2fd;
    border-color: #0d47a1;
}

QPushButton:pressed {
    background-color: #bbdefb;
}</string>
   </property>
   <property name="text">
    <string>‚úó Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
